stun/const.go

```diff
@@ -28,6 +29,14 @@ const (
 // NATType is the type of NAT described by int.
 type NATType int
 
+// NAT behavior type
+type NATBhvType int
+
+type NATBehavior struct {
+	MapBhvType NATBhvType // MappingBehaviorType
+	FltBhvType NATBhvType // FilteringBehaviorType
+}
```

Comment:
Whats up with these cryptic names? Are you paying per character?

Reply:
Thanks for response.   It's just some ridiculous thoughts: don't want it too long. Is it better use MType and FType?

Reply:
Should I move these to a test branch?  There may need much more adjust.

Reply:
The type is already called NATBehaviour, why can't you just call them FilteringType and BehaviourType?

Reply:
Good advice.